bin/ap_proto

```diff
@@ -0,0 +1,337 @@
+                            '. Time is assumed to be in UTC time zone. Used only to at'
+                            ' first invocation to intialize database.')
+        parser.add_argument('--start-visit-id', type=int, default=1,
+                            help='Starting visit ID. Used only to at first invocation'
+                            ' to intialize database. def: 1')
+
+        parser.add_argument('sources_file', help='Name of intput file with sources')
+        # add more command line options
+        self.args = parser.parse_args()
+
+        # configure logging
+        _configLogger(self.args.verbose)
+
+
+    def run(self):
+
+        # instantiate db interface
+        db = l1db.L1db(self.args.config)
+
+        # Initialize starting values from database visits table
+        last_visit = db.lastVisit()
+        if last_visit is None:
+            # database is empty
+            if self.args.start_time is None:
+                _LOG.error('Database is not initialized, --start-time option is required')
+                return 1
+        else:
+            self.args.start_visit_id = last_visit.visitId + 1
+            self.args.start_time = last_visit.visitTime + timedelta(seconds=self.args.interval)
+
+        # read sources file
+        _LOG.info("Start loading variable sources from %r", self.args.sources_file)
+        var_sources = numpy.load(self.args.sources_file)
+        _LOG.info("Finished loading variable sources, count = %s", len(var_sources))
+
+        # diaObjectId for last new DIA object, for variable sources we use their
+        # index as objectId, for transients we want to use ID outside that range
+        if last_visit is not None and last_visit.lastObjectId is not None:
+            self.lastObjectId = max(self.lastObjectId, last_visit.lastObjectId)
+        if self.lastObjectId < len(var_sources):
+            _LOG.error('next object id is too low: %s', nextObjectId)
+            return 1
+        _LOG.debug("lastObjectId: %s", self.lastObjectId)
+
+        # diaSourceId for last DIA source stored in database
+        if last_visit is not None and last_visit.lastSourceId is not None:
+            self.lastSourceId = max(self.lastSourceId, last_visit.lastSourceId)
+        _LOG.debug("lastSourceId: %s", self.lastSourceId)
+
+        # loop over visits
+        visitTimes = _visitTimes(self.args.start_time, self.args.interval, self.args.num_visits)
+        for visit_id, dt in enumerate(visitTimes, self.args.start_visit_id):
+
+            _LOG.info("Processing visit %s at %s", visit_id, dt)
+
+            # point telescope in random southern direction
+            pointing = generators.rand_sphere_xyz(1, -1)[0]
+            pointing_v = Vector3d(pointing[0], pointing[1], pointing[2])
+            ra = LonLat.longitudeOf(pointing_v).asDegrees()
+            decl = LonLat.latitudeOf(pointing_v).asDegrees()
+
+            _LOG.info("Pointing ra, decl = %s, %s; xyz = %s", ra, decl, pointing)
```

Comment:
Ah yes, I saw it as `"Pointing, ra, dec = %s, %s, %s", ra, decl, pointing` , what you have is correct then.